tests/docker/BUILD

```diff
@@ -252,10 +252,10 @@ container_push(
 container_push(
     name = "push_stamped_test",
     format = "Docker",
-    image = ":set_cmd",
+    image = "@io_bazel_rules_docker//testdata:files_in_layer_with_files_base",
```

Comment:
why do you need to use this one as base?
also, you should not need the @io_bazel_rules_docker

Reply:
I wanted an image that has multiple layers. The e2e tests push and then use docker pull on this image. This is expected to catch the scenario where the number of layers in the manifest and the image config disagree which would result in the docker pull failing

Reply:
Removed @io_bazel_rules_docker